app/models/manageiq/providers/hawkular/middleware_manager/event_catcher_mixin.rb

```diff
@@ -0,0 +1,67 @@
+# We use this to provide easy access to both the runner and specs
+module ManageIQ::Providers::Hawkular::MiddlewareManager::EventCatcherMixin
+  # extend ActiveSupport::Concern
+
+  # TODO: perhaps we need to block certain events.  Another way to do this would be to add a tag on events we want to
+  # suck into MiQ, and just fetch with the tag as criteria
+  INCLUDE_EVENTS = {'Hawkular Deployment' => '.*'}.freeze # some Hash of included events
+  EXCLUDE_EVENTS = {}.freeze # TODO: some Hash of exclusions for the included events
```

Comment:
@blomquisg OK, this one I'm not sure about.  What you see here is not filtering of MiQ events but rather filtering of Hawkular Events before they get put on the queue for MiQ.  Hawkular generates a variety of events and only a subset will be supported by, and be sent on to, MiQ.  So those pulled from the hawkular server then need to be thinned to be only those that should be exposed by the hawkular provider.  So the INCLUDE_EVENTS above is saying, for now, I only want to process 'Hawkular Deployment' events.  I think this is different than what you were saying but I'm not sure.  Also, perhaps what you are saying is I need to also support a further filter for events disabled at the MiQ level?